docs/intro/tutorial05.txt

```diff
@@ -457,7 +457,7 @@ class:
 
     def create_question(question_text, days):
         """
-        Creates a question with the given `question_text` published the given
```

Comment:
That would certainly do it. It took me a few reads to get the meaning when first reading through it.